src/boostact/Boostact.js

```diff
@@ -243,6 +245,12 @@ const updateNode = (currentNode) => {
         dom.addEventListener(eventType, newProps[name]);
       } else if (!name.startsWith("on") && typeof newProps[name] !== "function") {
         dom[name] = newProps[name];
+
+        if (name === "style") {
+          Object.keys(newProps[name]).forEach((prop) => {
+            dom["style"][prop] = newProps[name][prop];
```

Comment:
```suggestion
            dom[name][prop] = newProps[name][prop];
```
같은 style 문자라서 name 변수로 사용해도 될것같습니다.